tests/CLI_test/conftest.py

```diff
@@ -0,0 +1,60 @@
+import os
+import sys
+from pathlib import Path
+
+import pytest
+from _pytest.logging import LogCaptureFixture
+from cmd2.utils import StdSim
+from loguru import logger
+
+from FABulous.FABulous_CLI.FABulous_CLI import FABulous_CLI
+from FABulous.FABulous_CLI.helper import create_project, setup_logger
+
+
+def normalize(block: str):
+    """Normalize a block of text to perform comparison.
+
+    Strip newlines from the very beginning and very end  Then split into separate lines and strip trailing whitespace
+    from each line.
+    """
+    assert isinstance(block, str)
+    block = block.strip("\n")
+    return [line.rstrip() for line in block.splitlines()]
+
+
+def run_cmd(app, cmd):
+    """Clear out and err StdSim buffers, run the command, and return out and err"""
+    app.onecmd_plus_hooks(cmd)
+
+
+TILE = "LUT4AB"
+SUPER_TILE = "DSP"
+
+os.environ["FAB_ROOT"] = str(Path(__file__).resolve().parent.parent.parent / "FABulous")
+
+
+@pytest.fixture
+def cli(tmp_path):
+    projectDir = tmp_path / "test_project"
+    fabulousRoot = str(Path(__file__).resolve().parent.parent.parent / "FABulous")
+    os.environ["FAB_ROOT"] = fabulousRoot
+    create_project(projectDir)
+    setup_logger(0)
+    cli = FABulous_CLI(
+        writerType="verilog", projectDir=projectDir, enteringDir=tmp_path
```

Comment:
Should we also add a VHDL fixture?

Reply:
This is just testing is the CLI command working. Testing is fabric generation working I think should be done in testing for FABulous_API.